emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/CollectionServiceImpl.java

```diff
@@ -403,7 +403,7 @@ public boolean createCollection(DataGridCollectionAndDataObject collection) thro
 			IRODSFileFactory irodsFileFactory = irodsServices.getIRODSFileFactory();
 
 			IRODSFile newFile = irodsFileFactory.instanceIRODSFile(collection.getParentPath(), collection.getName());
-			irodsFileSystemAO.mkdir(newFile, false);
+			irodsFileSystemAO.mkdir(newFile, true);
```

Comment:
This changes the behavior of the function. Legacy applications may depend on that old behavior.

I think we need a new function.

Reply:
they could depend on it NOT making intermediate collections?

Reply:
Not just that, but possibly other things. It depends on what Jargon does when that option is set to true.

Adding a function that exposes the boolean parameter solves the issue without adding risk.

Reply:
fair enough.